### DISCUSSION TITLE
Contribution planning

### DISCUSSION THREAD

**Discussion by searge** [GeekOpsUA/aws_eks_karpenter_mf#3] [General]
### :vulcan_salute: @vitalibit, @gidra39 @oleksiihead

Які у вас плани, хто що собі вибрав [із тасок](https://github.com/orgs/GeekOpsUA/projects/1)?

**Comment by vitalibit** [comment]
> ### :vulcan_salute: @vitalibit, @gidra39 @oleksiihead
> 
> Які у вас плани, хто що собі вибрав [із тасок](https://github.com/orgs/GeekOpsUA/projects/1)?

Я з @gidra39 візьмемо EKS

**Reply by searge** [reply]
Ok

**Comment by searge** [comment]
@mirik12 може ти візьмешся за [IAM](https://github.com/orgs/GeekOpsUA/projects/1?pane=issue&itemId=51803463)?

**Comment by searge** [comment]
@GeekOpsUA/cloud-devs як справи по проекту :relaxed:?

**Reply by vitalibit** [reply]
закінчую підготовку :)

**Reply by searge** [reply]
:mechanical_arm:
